.appveyor.yml

```diff
@@ -16,6 +16,7 @@ image:
 environment:
   # Create expected SHELL variable for pipenv.
   SHELL: "windows"
+  CTEST_OUTPUT_ON_FAILURE: "1"
```

Comment:
Is this one used automagically? Or is it a leftover of some previous commit?

Reply:
My hope is that the above line sets the CTEST_OUTPUT_ON_FAILURE environment variable which is understood by ctest to output more than just "test failed".